README.md

```diff
@@ -116,6 +117,7 @@ Source included in lua directory
 
 * luacurl (rename library to c-luacurl -- there is an additional lua module for this library)
 * luatask (rename library to c-task -- there is an additional lua module for this library)
+* lyaml (rename library to c-yaml -- there is an additional lua module for this library)
```

Comment:
Does this work without the rename? IIRC the c-luacurl and c-task thing is b/c those two are written exclusively in c and I added a lua wrappers but wanted to keep the original lua module name (honestly both of those lua wrappers could constitute entirely new modules, but again, weird decisions long ago).

In this case the c library is just `yaml` and the lua library is `lyaml`, so I don't think there's a conflict.

Reply:
So I actually have a bug in the current PR that I introduced when cleaning it up. The Lua code in lyaml has `require 'yaml'` in init.lua, and it seems that this causes it to try to load from yaml.dll, which is libyaml, not the C portion of lyaml. The lyaml C library registers itself as "yaml" when calling luaL_register. So I need to change lyaml's init.lua to ```require 'c-yaml'``` to get it to point to the right .dll file.

Lua's module loading is a bit of a mystery to me, but do you still think this could be simplified in light of that and/or have a different suggestion to avoid that conflict?

Reply:
Have you changed the code since this comment? It looks like right now we have lyaml (lua) -> c-yaml (c lyaml library) -> yaml (libyaml) . . . and I would expect that to work.

Reply:
Yep, what's currently uploaded should be functional - I changed lyaml's init.lua to require c-yaml instead of yaml to match the renamed library.